examples/options/src/program_options.cpp

```diff
@@ -38,7 +38,9 @@ program_options::program_options(
 
     // Print a help message if the user asked for it.
     if (vm.count("help")) {
-        std::cout << m_desc << std::endl;
+        std::ostringstream oss;
+        oss << m_desc;
+        TRACCC_INFO(oss.str());
```

Comment:
I would actually say this is a rare exception where it's better _not_ to use the logger; the additional time, log level, etc. is not necessary to show the user the runtime flags.

Reply:
:+1: